lstmcpipe/lstmcpipe_start.py

```diff
@@ -221,7 +221,7 @@ def main():
     if "train_pipe" in stages_to_run:
 
         job_from_train_pipe = batch_train_pipe(
-            lstmcpipe_config,
+            lstmcpipe_config['stages']['train_pipe'],
```

Comment:
(....) and here we will just need to do a `lstmcpipe_config['stages']['train_pipe'][0]`

Reply:
So, no.
If we generate and pass a list here, the assumption will be that everything in that list will be handled, not just the first item. So either the stage `train_pipe` is implemented in a way to handle the complete list (see above comment, not so easy imo), or we pass a single dict - matching the stage logic.

Reply:
pass a single dict then, imo